crates/languages/src/dart/indents.scm

```diff
@@ -1,7 +1,45 @@
 [
-  (if_statement)
-  (for_statement)
-] @indent
+  (class_body)
+  (function_body)
+  (function_expression_body)
+  (declaration
+    (initializers))
+  (switch_block)
+  (formal_parameter_list)
+  (formal_parameter)
+  (list_literal)
+  (return_statement)
+  (arguments)
+  (try_statement)
+] @indent.begin
 
-(_ "{" "}" @end) @indent
-(_ "(" ")" @end) @indent
+(switch_block
+  (_) @indent.begin
+  (#set! indent.immediate 1)
+  (#set! indent.start_at_same_line 1))
+
+[
+  (switch_statement_case)
+  (switch_statement_default)
+] @indent.branch
+
+[
+  "("
+  ")"
+  "{"
+  "}"
+  "["
+  "]"
+] @indent.branch
+
+"}" @indent.end
+
+(return_statement
+  ";" @indent.end)
+
+(break_statement
+  ";" @indent.end)
+
+(comment) @indent.ignore
+
+(if_statement) @indent.auto
```

Comment:
This looks like a file that's been copied from elsewhere. maybe nevoid? Zed doesn't use this same indent syntax. Can you undo the changes to this file?

Reply:
I kept neovim as base and made few minor changes. I've reverted the file contents now.